docs/src/docs/snippets/gradle/kotlin/build.gradle.kts

```diff
@@ -221,3 +221,11 @@ graalvmNative {
     }
 }
 // end::specify-metadata-version-for-library[]
+
+// tag::include-metadata[]
+var reachabilityMetadata = tasks.register<org.graalvm.buildtools.gradle.tasks.ReachabilityMetadataCopyTask>("reachabilityMetadata")
+
+tasks.withType<Jar> {
```

Comment:
:x: This is inconsistent with the Groovy version. Either you want to update _only_ the jar, and this should be:

```
tasks.named("jar", Jar) {
...
}
```
or, you want to update all the jar files (I wouldn't recommend) and this should be:

```
tasks.withType<Jar>().configureEach {
    ...
}
```

(note the `configureEach` to ensure laziness)

Reply:
Thanks. I've not used the Kotlin DSL much so this as all a bit alien to me.